internal/proxy/options.go

```diff
@@ -19,6 +19,7 @@ import (
 // Options are configuration options that can be set by Environment Variables
 // Port - int -  port to listen on for HTTP clients
 // ProviderURLString - the URL for the provider in this environment: "https://sso-auth.example.com"
+// ProxyProviderURLString - the internal URL for the provider in this environment: "https://sso-auth-int.example.com"
```

Comment:
If this field is truly intended as an internal URL, would it be clearer if the name reflected that? I struggle a little bit with the names `ProviderURLString` and `ProxyProviderURLString` being not meaningfully different as far as I can tell.

Reply:
thought here is that the `sso_proxy` service is using this one, the other is for the `client`.  but then naming is not my strong suit